Backend/routes/routes.py

```diff
@@ -0,0 +1,26 @@
+from fastapi import APIRouter
+from Backend.model.model import Ngo
+from Backend.config.database import collection
+from Backend.schemas.schema import ngo_addition_serializer
+from bson import ObjectId
+
+ngo_addition_router = APIRouter()
```

Comment:
we can rename this router object to ngo_router as it is not restricted only for addition